src/shared/remoteSourceTrackingService.ts

```diff
@@ -456,25 +462,30 @@ export class RemoteSourceTrackingService extends ConfigFile<RemoteSourceTracking
         return;
       }
 
-      if (outstandingSourceMembers.size < 20) {
-        this.logger.debug(
-          outstandingSourceMembers.size < 20
-            ? `Still looking for SourceMembers: ${Array.from(outstandingSourceMembers.keys()).join(',')}`
-            : `Still looking for ${outstandingSourceMembers.size} Source Members`
-        );
+      if (someResultsReturned && consecutiveEmptyResults >= CONSECUTIVE_EMPTY_POLLING_RESULT_LIMIT) {
```

Comment:
Does this need to have a comment about it because it's somewhat misleading?  The limit const is actually a polling timeout in seconds.

Reply:
no, it's really the number of requests (since the delay between them is 1s).  I can make that logic clearer up when this is calculated.